sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/implementation/pipeline/TelemetryItemExporter.java

```diff
@@ -131,18 +132,9 @@ CompletableResultCode internalSendByBatch(TelemetryItemBatchKey telemetryItemBat
     private static List<ByteBuffer> serialize(List<TelemetryItem> telemetryItems) {
         try {
             if (logger.canLogAtLevel(LogLevel.VERBOSE)) {
-                try (StringWriter debug = new StringWriter()) {
-                    for (int i = 0; i < telemetryItems.size(); i++) {
-                        JsonWriter jsonWriter = JsonProviders.createWriter(debug);
-                        telemetryItems.get(i).toJson(jsonWriter);
-                        jsonWriter.flush();
-
-                        if (i < telemetryItems.size() - 1) {
-                            debug.write('\n');
-                        }
-                    }
-                    logger.verbose("sending telemetry to ingestion service:{}{}", System.lineSeparator(), debug);
-                }
+                String json = toJson(telemetryItems);
+                String jsonWithoutIKeys = removeIKeys(telemetryItems, json);
+                logger.verbose("sending telemetry to ingestion service:{}{}", System.lineSeparator(), jsonWithoutIKeys);
```

Comment:
nit: Don't use System.lineSeparator() in logs, it can be unsafe and `ClientLogger` will remove them anyways.

Reply:
From what I observe, the line separator is not removed in this example by using `ClientLogger` with Logback.

With a System.lineSeparator(), we are more confident that the user will see the line separator in his environment.